Luke_Smith_Portfolio.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```

Comment:
Please change the name of this file to "index.html"
Conventionally, the name of the main page of a website is index.html